src/fontra/views/fontinfo/panel-development-status-definitions.js

```diff
@@ -0,0 +1,325 @@
+    this.panelElement.focus();
+  }
+
+  async newStatusDef(statusDef = undefined) {
+    const statusFieldDefinitions =
+      this.fontController.customData["fontra.sourceStatusFieldDefinitions"];
+    if (!statusFieldDefinitions) {
+      // TODO: This does not work, yet.
+      // If there is no "fontra.sourceStatusFieldDefinitions" in customData.
+      this.fontController.customData["fontra.sourceStatusFieldDefinitions"] = [];
+    }
+    const nextStatusValue = !statusFieldDefinitions
+      ? 0
+      : statusFieldDefinitions
+          .map((statusDef) => statusDef.value)
+          .sort()
+          .pop() + 1;
+
+    // fix duplicates
+    if (
+      statusDef &&
+      statusFieldDefinitions.some((statusDef) => statusDef.value === statusDef.value)
+    ) {
+      // Status definition with value already exists,
+      // changed value to next available value.
+      statusDef = {
+        ...statusDef,
+        value: nextStatusValue,
+      };
+    }
+
+    // fix isDefault
+    if (statusDef && statusFieldDefinitions.some((statusDef) => statusDef.isDefault)) {
+      // Status definition with isDefault already exists,
+      // changed value to false.
+      delete statusDef["isDefault"];
+    }
+
+    if (!statusDef) {
+      const defaultStatuses =
+        defaultStatusFieldDefinitions["fontra.sourceStatusFieldDefinitions"];
+      statusDef = defaultStatuses.find(
+        (statusDef) => statusDef.value == nextStatusValue
+      );
+    }
+
+    if (!statusDef) {
+      // No status definition provided or found.
+      // Use default.
+      statusDef = {
+        color: [1, 0, 0, 1],
+        label: `Status definition ${nextStatusValue}`,
+        value: nextStatusValue,
+      };
+    }
+
+    const undoLabel = `add status definition ${statusDef.value} '${statusDef.label}'`;
+    const root = { customData: this.fontController.customData };
+    const changes = recordChanges(root, (root) => {
+      root.customData["fontra.sourceStatusFieldDefinitions"].push(statusDef);
+    });
+    if (changes.hasChange) {
+      this.postChange(changes.change, changes.rollbackChange, undoLabel);
+      this.setupUI();
+    }
+  }
+}
+
+function getStatusFieldDefinitions(fontController) {
+  const statusFieldDefinitions =
+    fontController.customData["fontra.sourceStatusFieldDefinitions"];
+  if (statusFieldDefinitions) {
+    return statusFieldDefinitions;
+  }
+  return [];
+}
+
+addStyleSheet(`
+.fontra-ui-font-info-status-definitions-panel-status-def-box {
+  background-color: var(--ui-element-background-color);
+  border-radius: 0.5em;
+  padding: 1em;
+  cursor: pointer;
+  display: grid;
+  grid-template-rows: auto auto;
+  grid-template-columns: max-content max-content max-content auto;
+  grid-row-gap: 0.1em;
+  grid-column-gap: 1em;
+}
+
+.fontra-ui-font-info-status-definitions-panel-status-def-box-delete {
+  justify-self: end;
+  /*align-self: start;*/
+}
+
+.fontra-ui-font-info-status-definitions-panel-status-def-box-color {
```

Comment:
Suggest to add `-input` to this already long name. It took me a few seconds to realize this is about the color picker, and not about an actual color...

Reply:
OK. Did it.